test/units/modules/system/test_iptables.py

```diff
@@ -828,3 +828,82 @@ def test_iprange(self):
             '--dst-range',
             '10.0.0.50-10.0.0.100'
         ])
+
+    def test_insert_rule_with_wait(self):
```

Comment:
This function is unnecessary, it has not been on PR https://github.com/ansible/ansible/pull/71496.